### PR TITLE
(WIP) Doc + cleanup PrestaShopLogger class

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrading the PHPDoc comments of this class, using the latest PS standards and also applying the PS standards to some of the code. Also getting the Franglais out of the comments, getting rid of things the Symfony code sniffer nags about, etc. It might have some additional `todo` tags, depending on whether the code was comprehensible or not. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

Warning: this PR also has a code cleanup. Please check this first before merging!

### CODE REVIEW

File: classes/PrestaShopLogger.php [hunk 1]
```diff
@@ -163,7 +169,7 @@ public static function eraseAllLogs()
      *
      * @return true if exists
      */
-    protected function _isPresent()
+    protected function isPresent()
```
Comment (julienbourdeau) [inline classes/PrestaShopLogger.php]: You cannot modify the name of protected or public method, you have to at least alias it. It breaks travis build.

Comment (firstred) [inline classes/PrestaShopLogger.php]: Ahh, so that's what I have forgotten

Comment (aleeks) [inline classes/PrestaShopLogger.php]: Hello @firstred 
Can you change it and rebase your branch please ?!
`!$log->_isPresent()` vs `function isPresent()`
Thank you!

**Comment by firstred** [comment]
Rebased

**Comment by aleeks** [comment]
Thank you!

**Pull request merged**